src/keymap.cpp

```diff
@@ -740,15 +740,29 @@ void KeyMap::handle_action(const std::string& action, const std::string& params)
 
 std::vector<MacroCmd> KeyMap::parse_operation_sequence(const std::string& line)
 {
-	const auto operations = utils::tokenize_quoted(line, ";");
+	const auto words = utils::tokenize_quoted(line, " ");
+
+	std::vector<std::string> tokens;
+	for (auto word : words) {
+		if (word.size() > 1 && word.back() == ';') {
+			// Found a token with no space between the command and the
+			// semicolon; let's split it in two tokens
```

Comment:
It looks like this does not handle semicolons when there is no space around them.
A configuration like `macro x open;open-in-browser;quit` seems to work on the current master branch but no longer works after applying this PR.